site/content/en/docs/tasks/run_python_jobs.md

```diff
@@ -326,7 +325,7 @@ Use:
 "kubectl get jobs" to see jobs
 ```
 
-{{% include "python/sample-mpijob.py" "python" %}}
+{{< include "examples/python/sample-mpijob.py" "python" >}}
```

Comment:
This is just a confirmation. Why do we need to use a different format? 
`{{< include file="examples/<your-filepath>" lang="<your-code-language>" >}}` vs `{{< include "examples/<your-filepath>" "<your-code-language>" >}}`

Can we use the same format across the whole of the document to improve maintainability?

Reply:
I thought about this for a bit. I don't have a good reason to use two different formats. I just used two different formats in the md files because my shortcode implementation supports. For the goal of being maintainable, I am very happy to only keep one of them inside the documents. 

`{{< include file="examples/<your-filepath>" lang="<your-code-language>" >}}` used named parameter. It is verbose but clear on meaning of the parameters. Also the order of the parameters doesn't matter anymore. 

The second one, `{{< include "examples/<your-filepath>" "<your-code-language>" >}}`, is more concise. However, we have to pass the parameters in the correct order. 

which one would you want to keep?

Reply:
I like the verbosity, but let's be consistent.

Reply:
Let's merge as is for now

/lgtm
/approve
/hold cancel